pkgs/by-name/ch/chirpstack/package.nix

```diff
@@ -0,0 +1,267 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nixosTests,
+  yarnConfigHook,
+  yarnBuildHook,
+  fetchYarnDeps,
+  rustPlatform,
+  nix-update-script,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.